src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -990,14 +992,6 @@ export default {
 	background: linear-gradient(0deg, gray, white 36px);
 }
 
-.network-connection-state {
-	position: absolute;
-	bottom: 3px;
-	right: 12px;
-	width: 32px;
-	height: 32px;
-}
```

Comment:
![Bildschirmfoto von 2021-11-17 22-18-54](https://user-images.githubusercontent.com/213943/142284148-60c3d133-780e-4fe2-b440-3f4a2b500d72.png)

The spacing doesn't look right to me. the icon is more to the top and too close to the right